servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -89,6 +90,32 @@ public FileInfo generateClientFileInfo(String path) {
   }
 
   /**
+   * Reinitializes the inode file.
+   */
+  public void reinit() {
```

Comment:
I thought about that. But I ended up with `reinit` because this is only called when a file is not completed. For `reset` I feel it can erase the old file. What do you think?